apache/arrow
# ARROW-5371: [Release] Add tests for dev/release/00-prepare.sh

Test is implemented by Ruby and test-unit.

How to test:

  * Clone the current Git repository for test
  * Run `dev/release/00-prepare.sh` in test test repository
  * Check diff in `git log -n 1 -p`
